javascript/multi-user-sub.js

```diff
@@ -0,0 +1,3 @@
+channel.subscribe('update', msg => {
+  document.applyUpdate(msg.data);
```

Comment:
Same here

Reply:
Why?  Developers can assume there is a document object.